oximeter/oximeter/schema/switch-data-link.toml

```diff
@@ -263,6 +284,10 @@ description = "Revision number of the switch the link is on"
 type = "string"
 description = "Serial number of the switch the link is on"
 
+[fields.switch_slot]
+type = "u32"
```

Comment:
I think we could get away with a `u8` here.

Reply:
Going with u16 to match.